### PR TITLE
Simplified Conditionals

### PR DESCRIPTION
**Pull request by lukasberglund**
This stuff should have been in my code review but I didn't think about it until yesterday. Goes to show I should put more thought into code reviews. I'll be better about it next time.

### CODE REVIEW

**Comment by ayazhafiz** [comment]
> Goes to show I should put more thought into code reviews. I'll be better about it next time.

No worries, this is an iterative process and the point is to be able to fixup code that we write earlier :)

File: src/Parser.hs [hunk 1]
```diff
@@ -126,35 +126,28 @@ parseExpr precedence delims tokens =
...
+  let terminator:afterTerminator = afterIf
   (else', afterConditional) <-
-    case symbol $ head elseTokens of
-      Symbol.Else -> do
-        (exprs, afterExpressions) <- parseExpressionList (tail elseTokens)
-        Right (Just exprs, afterExpressions)
-      _ -> Right (Nothing, afterElseIf)
-  return (Grammar.Conditional if' then' elseIfs else', afterConditional)
+    case symbol $ head afterTerminator of
+      Symbol.ElseIf -> do
```
Comment (ayazhafiz) [inline src/Parser.hs:133]: I am thinking we should change the syntax of `elseif` to be `else if`, which would make the way we express the grammar fall in line with the syntax of the language.

Comment (lukasberglund) [reply src/Parser.hs:133]: True. Let's do it. So `else if` would be syntactic sugar for `else { if`

Comment (lukasberglund) [reply src/Parser.hs:133]: Let's add it as an issue

Comment (ayazhafiz) [reply src/Parser.hs:133]: Yep

**Pull request merged**